src/freenas/etc/find_alias_for_smtplib.py

```diff
@@ -2,133 +2,125 @@
 import email
 import email.parser
 import json
-import os
-import re
 import requests
 import sys
-import syslog
 
 from truenas_api_client import Client
 
-ALIASES = re.compile(r'^(?P<from>[^#]\S*?):\s*(?P<to>\S+)$')
-
 
 def do_sendmail(msg, to_addrs=None, parse_recipients=False):
+    to_addrs = ["root"] if to_addrs is None else to_addrs
+    if to_addrs is None and not parse_recipients:
```

Comment:
This is an impossible condition right? `to_addrs` will never be `None` here

Reply:
`to_addrs` is a keyword argument passed to this function so it can be `None`.

Reply:
oh you're right, i'll fix

Reply:
I pushed a new commit. We actually don't even use the `find_alias*` scripts on SCALE so we can remove all this stuff.